config/routes.rb

```diff
@@ -18,496 +20,502 @@
     delete "/sign_out", to: "devise/sessions#destroy"
   end
 
-  require "sidekiq/web"
-  require "sidekiq_unique_jobs/web"
-  require "sidekiq/cron/web"
+  # [@forem/delightful] - all routes are nested under this optional scope to
+  # begin supporting i18n.
+  scope "(/locale/:locale)", defaults: { locale: nil } do
```

Comment:
There are several routes here that we could move outside of the `/locale/:locale` scope because we don't need them internationalized. I figured by starting with them all nested, we can unnest them over time. As long as the default URLs still work, it doesn't hurt to have them here.